### PR TITLE
Price, date calculations

### PR DESCRIPTION
**Pull request by aule2000**
Created price calculation by distance and date calculation by isExpress value

### CODE REVIEW

**Review by Filari02 — APPROVED** [review]

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java [hunk 1]
```diff
@@ -97,6 +101,15 @@ public Order createOrder(@RequestBody Order order)
             addressController.createAddress(order.getSourceAddress());
         }
 
+        Date date = new Date();
+        order.setDate(date);
+
+        order.setEstimatedArrivalTime(orderService.calculateArrivalTime(order.getIsExpress()));
+        if (order.getSize() == null){
+            throw new CustomException("Box size is null");
```
Comment (Ugniusst) [inline Backend/src/main/java/lt/vu/ads/controllers/OrderController.java:119]: While getting orders from database, there shouldn't be any null sizes. So is it important to check here?

Comment (aule2000) [reply Backend/src/main/java/lt/vu/ads/controllers/OrderController.java:119]: If you talk about order.getIsExpress value, so by default it is false. So, I think this value will never be null.

**Review by Ugniusst — COMMENTED** [review]
If I understood well, order code is generated randomly without taking into account order id or else. If so, it should be checked if there are no duplicates.

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java (hunk 1 above)
Comment (Ugniusst) [reply Backend/src/main/java/lt/vu/ads/controllers/OrderController.java:119]: No, Im talking about order.getSize() == null. Will it be null in get controller (not post)

File: Backend/src/main/java/lt/vu/ads/constants/PriceConstants.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package lt.vu.ads.constants;
+
+public class PriceConstants {
+
+    public static double PRICE_PER_KM = 0.2;
+    public static double EXPRESS_PRICE_ADDITION = 3;
+
+    public static double S_SIZE_PRICE = 1.3;
+    public static double M_SIZE_PRICE = 1.7;
+    public static double L_SIZE_PRICE = 2;
```
Comment (Tomas-Per) [inline Backend/src/main/java/lt/vu/ads/constants/PriceConstants.java]: Maybe add `final` for the constants

File: Backend/src/main/java/lt/vu/ads/controllers/OrderController.java (hunk 1 above)
Comment (aule2000) [reply Backend/src/main/java/lt/vu/ads/controllers/OrderController.java:119]: To check if the value is null is worth for security reasons. For example, if the UI does not have default value.

**Review by Ugniusst — APPROVED** [review]

**Review by Tomas-Per — APPROVED** [review]

**Review by Foteno — APPROVED** [review]
Looks good

**Pull request merged**